action.yaml

```
@@ -67,4 +67,4 @@ runs:
     - name: Run tox
       shell: bash
       working-directory: ${{ inputs.working-directory }}
-      run: tox -e ${{ inputs.tox-envs }} ${{ inputs.tox-posargs }}
+      run: tox -e ${{ inputs.tox-envs }} -- ${{ inputs.tox-posargs }}
```

Oh, I haven't updated this for tox 4... should this be:

```suggestion
      run: tox run -e ${{ inputs.tox-envs }} -- ${{ inputs.tox-posargs }}
```

---

Oh, indeed, good catch.